hack/debug.sh

```diff
@@ -545,13 +545,17 @@ case "$1" in
 	    echo "  [from master]"
 	    echo "    $0"
 	    echo "  Gathers data on the master and then connects to each node via ssh"
+	    echo "    $0 --master"
+	    echo "  Runs only master debugging on local machine"
 	    echo ""
 	    echo "  [from any other machine]"
 	    echo "    $0 MASTER-NAME"
 	    echo "  Connects to MASTER-NAME via ssh and then connects to each node via ssh"
+	    echo "    $0 --node"
+	    echo "  Runs only node debugging on local machine"
```

Comment:
This requires some setup to have happened doesn't it? Like, you need to have the meta/ directory from a --master run?

If we're going to document the "manual" usage, it should be separate from the existing documentation of the "automatic" usage

Reply:
you're right, but this is just the script usage, i guess a more detailed information should be in the official documentation or in the README file

Reply:
"From any other machine" there doesn't mean "from a node". It means "from a random machine outside the cluster". So it doesn't make sense to document `--master` up above but `--node` here.

The existing documentation isn't just enumerating command-line options, it's explaining the two ways you can run the script: you can run it on the master, or you can run it from any machine that can ssh to the master and the nodes. What you're documenting is a third way of running it, but you're mixing the documentation of that mode in with the documentation of the other two modes.

Reply:
> i guess a more detailed information should be in the official documentation

basically, this *is* the official documentation